sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfigurationFacadeEjb.java

```diff
@@ -254,6 +257,35 @@ public boolean isFeatureDisabled(FeatureType featureType) {
 		return em.createQuery(cq).getSingleResult() > 0;
 	}
 
+	@Override
+	@SuppressWarnings("unchecked")
+	public boolean isPropertyValue(FeatureType featureType, FeatureTypeProperty property, boolean expectedPropertyValue) {
+
+		if (!Arrays.asList(featureType.getSupportedProperties()).contains(property)) {
+			throw new IllegalArgumentException("Feature type " + featureType + " does not support property " + property + ".");
+		}
+
+		if (!Boolean.class.isAssignableFrom(property.getReturnType())) {
+			throw new IllegalArgumentException(
+				"Feature type property " + property + " does not have specified return type " + Boolean.class.getSimpleName() + ".");
+		}
+
+		CriteriaBuilder cb = em.getCriteriaBuilder();
+		CriteriaQuery<Object> cq = cb.createQuery(Object.class);
+		Root<FeatureConfiguration> root = cq.from(FeatureConfiguration.class);
+
+		cq.where(cb.and(cb.equal(root.get(FeatureConfiguration.FEATURE_TYPE), featureType)));
+		cq.select(root.get(FeatureConfiguration.PROPERTIES));
+
+		Map<FeatureTypeProperty, Object> properties = (Map<FeatureTypeProperty, Object>) em.createQuery(cq).getSingleResult();
+
+		if (properties != null && properties.containsKey(property)) {
```

Comment:
Initialize only one return value, suppress warning locally on the line that causes it, not for the whole method.
Before if-else:
```
final boolean result;
```

Reply:
Moved the annotation; can you please explain the advantage of initialising a result variable? It doesn't really make the code more readable in my opinion.

Reply:
It's easier to debug, review and appendable when you don't have several `return` statements.